packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -196,8 +198,8 @@ ${buildNFTAnnotations(edge || !copyEngine, clientEngineType, binaryTargets, rela
 
     const modelEnums: string[] = []
     const modelEnumsAliases: string[] = []
-    for (const enumType of this.dmmf.schema.enumTypes.model ?? []) {
-      modelEnums.push(new Enum(enumType, false).toTS())
+    for (const enumType of this.dmmf.datamodel.enums) {
+      modelEnums.push(new Enum({ name: enumType.name, values: enumType.values.map(({ name }) => name) }, false).toTS())
```

Comment:
let's maybe extract a small function to convert from `DatamodelEnum` to `SchemaEnum`